### ISSUE TITLE
RenewSliver fails with old SFA-based aggregates like MyPLC

### ISSUE THREAD

**Issue by tcmitchell** [GENI-NSF/geni-tools#65]
Omni now sends a timezone on the date/time passed in RenewSliver to comply with the GENI AM API requirement that they be [RFC 3339](http://tools.ietf.org/html/rfc3339) compliant. Older SFA-based aggregates like the MyPLC aggregates deployed in mesoscale GENI fail when the timezone is present with the error message:
```
Fault 111: "Internal API error: can't compare offset-naive and offset-aware date times"
```
Provide a way for omni to work with new aggregates while being backward compatible with older aggregates.

 _Imported from trac ticket [#65](http://trac.gpolab.bbn.com/gcf/ticket/65),  created by tmitchel on 09-02-2011 at 14:19, last modified: 09-02-2011 at 14:37_
   CCing: nriga@..., ahelsing@..., sedwards@...

**Comment by tcmitchell** [comment]
Here is the full omni trace when this happens:
```

$ src/omni.py -c ~/.gcf/omni-gec9 -a http://myplc.gpolab.bbn.com:12346 renewsliver pgenitz1 20110903T10:00
INFO:omni:Loading config file /home/tmitchel/.gcf/omni-gec9
INFO:omni:Using control framework pgeni
INFO:omni:Slice urn:publicid:IDN+pgeni.gpolab.bbn.com+slice+pgenitz1 expires within 1 day on 2011-09-03 12:00:00 UTC
INFO:omni:Renewing Sliver urn:publicid:IDN+pgeni.gpolab.bbn.com+slice+pgenitz1 until 2011-09-03 10:00:00+00:00 (UTC)
ERROR:omni.protogeni:Call for Renew Sliver urn:publicid:IDN+pgeni.gpolab.bbn.com+slice+pgenitz1 on http://myplc.gpolab.bbn.com:12346 failed. Server says: <Fault 111: "Internal API error: can't compare offset-naive and offset-aware datetimes">
WARNING:omni:Failed to renew sliver urn:publicid:IDN+pgeni.gpolab.bbn.com+slice+pgenitz1 on urn (http://myplc.gpolab.bbn.com:12346) <Fault 111: "Internal API error: can't compare offset-naive and offset-aware datetimes">
INFO:omni: ------------------------------------------------------------
INFO:omni: Completed renewsliver:

  Options as run:
		aggregate: http://myplc.gpolab.bbn.com:12346
		configfile: /home/tmitchel/.gcf/omni-gec9
		framework: pgeni
		native: True

  Args: renewsliver pgenitz1 20110903T10:00

  Result Summary: Slice urn:publicid:IDN+pgeni.gpolab.bbn.com+slice+pgenitz1 expires within 1 day(s) on 2011-09-03 12:00:00 UTC
Failed to renew sliver urn:publicid:IDN+pgeni.gpolab.bbn.com+slice+pgenitz1 on urn (http://myplc.gpolab.bbn.com:12346) <Fault 111: "Internal API error: can't compare offset-naive and offset-aware datetimes">
 
INFO:omni: ============================================================
```

_Trac comment by tmitchel (github user: tcmitchell) on 09-02-2011 at 14:28_

**Comment by tcmitchell** [comment]
Fixed in [c46420eacc5d89f93345c7f461aa79b42d30a132](https://github.com/GENI-NSF/geni-tools/commit/c46420eacc5d89f93345c7f461aa79b42d30a132)

Added a "--no-tz" command line option to to tell omni to avoid passing a timezone on the RenewSliver call.

_Trac comment by tmitchel (github user: tcmitchell) on 09-02-2011 at 14:33_
